glommio/src/executor/placement/mod.rs

```diff
@@ -67,37 +67,35 @@ use std::{
 type Result<T> = crate::Result<T, ()>;
 
 #[cfg(doc)]
-use super::{LocalExecutor, LocalExecutorPoolBuilder};
+use super::{LocalExecutor, LocalExecutorBuilder, LocalExecutorPoolBuilder};
 
 /// Specifies a policy by which [`LocalExecutorPoolBuilder`] selects CPUs.
 ///
-/// `Placement` is use to bind [`LocalExecutor`]s to a set of CPUs via
-/// preconfigured policies designed to address a variety of use cases.  The
-/// default is `Unbound`.
+/// `PoolPlacement` is used to bind [`LocalExecutor`]s to a set of CPUs via
+/// preconfigured policies designed to address a variety of use cases.
 ///
 /// ## Example
 ///
-/// Some `Placement`s allow manually filtering available CPUs via a [`CpuSet`],
-/// such as `MaxSpread`.  The following would place shards on four CPUs
-/// (a.k.a.  hyper-threads) that are on NUMA node 0 and have an even numbered
-/// package ID according to their [`CpuLocation`]. The selection aims to achieve
-/// a high degree of separation between the CPUs in terms of machine topology.
-/// Each [`LocalExecutor`] would be bound to a single CPU.
+/// Some `PoolPlacement`s allow manually filtering available CPUs via a
+/// [`CpuSet`], such as `MaxSpread`.  The following would place shards on four
```

Comment:
This is only a variant of `PoolPlacement` right? Should we reword this to mention `Fenced` or `Exact`?

Reply:
This comment describes the behavior of the following code snippet, so I don't think it's necessary to mention the others (all the variants are just below and individually documented).